zmon-controller-app/src/main/java/org/zalando/zmon/config/FeedbackMetadataProperties.java

```diff
@@ -0,0 +1,14 @@
+package org.zalando.zmon.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "zmon.feedback")
+public class FeedbackMetadataProperties {
+    private String url;
+
+    public Boolean feedbackEnabled = false;
```

Comment:
Don't you need a setter for that to work? @alexkorotkikh

Reply:
That's true

https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-typesafe-configuration-properties

Reply:
please make it `private final` and use a primitive type.